src/app/frontend/uicomponents/helpsection/helpsection_directive.js

```diff
@@ -0,0 +1,31 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import HelpSectionController from './helpsection_controller';
+
+/**
+ * Returns a directive definition object for the reusable sections with help.
+ * @return {!angular.Directive} the directive definition
+ */
+export default function() {
+  return {
+    bindToController: true,
+    controller: HelpSectionController,
+    controllerAs: 'ctrl',
+    scope: {},
+    template:
+        `<ng-transclude class="kd-help-section" layout="row" layout-xs="column"> </ng-transclude>`,
```

Comment:
Dont hesitate to put this into a html file. We have a nice processing pipeline of html code in html files, which does not catch inline templates :)